Jenkinsfile-Agent-Test-3

```diff
@@ -67,7 +67,7 @@ pipeline {
                 sh './mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build'
             }
         }
-        stage('Test Cases Report (161)') {
+        stage('Test Cases Report (214)') {
```

Comment:
Could you make it place at Workload2-Group2?

Reply:
I am glad to.